Codegen/test/resources/input/TypeCobol/CopyReplace4Colon/CopyReplace4Colon.rdz.cbl

```diff
@@ -0,0 +1,10 @@
+﻿       IDENTIFICATION DIVISION.
```

Comment:
This test, CopyReplace4Colon4.rdz.cbl and CopyReplace4Colon6.rdz.cbl are almost identical.
Can you merge them into a single one.

Reply:
Ok I will

Reply:
Done by 
1434b65  with the mistake on the target issue